### PR TITLE
server timing doc

### PR DESCRIPTION
**Pull request by i110**
Adds docs about `server-timing` directive and Proxy Timings access log extensions.

### CODE REVIEW

File: srcdoc/configure/server_timing_directives.mt [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    default => 'OFF',
+    since   => '2.3',
+    desc    => <<'EOT',
+A flag indicating how <code>server-timing</code> header and trailer should be served.
+EOT
+)->(sub {
+?>
+
+<p>
+If the argument is <code>ON</code>, H2O adds the <code>server-timing</code> header and trailer (if possible) to the response. The header is always added, but the trailer isn't if all of the following conditions are met.
```
Comment (tmaesaka) [inline srcdoc/configure/server_timing_directives.mt]: Sanity checking, do you really mean this?
> if _**all**_ of the following conditions

Comment (tmaesaka) [reply srcdoc/configure/server_timing_directives.mt]: D'oh. I overlooked the </p> on L35. SGTM.

Comment (tmaesaka) [inline srcdoc/configure/server_timing_directives.mt]: In other references to the possible values (e.g. L38, L42), you use the word `value`. Would it help to keep the narrative consistent?

File: srcdoc/configure/server_timing_directives.mt [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+<p>
+If the value is <code>ENFORCE</code>, H2O forces the response to use chunked encoding by removing <code>content-length</code> header.
+</p>
+
+<p>
+If the value is <code>OFF</code>, the feature is disabled.
+</p>
+
+<p>
+The <code>server-timing</code> header and trailer include the following metrics. For now, all metrics have the <code>dur</code> attribute whose values are exactly equal to one of Access Log Extensions with similar names. To get the meaning of each <code>dur</code> attribute, see <a href="https://h2o.examp1e.net/configure/access_log_directives.html#access-log"><code>access-log</code></a>.
```
Comment (tmaesaka) [inline srcdoc/configure/server_timing_directives.mt]: Tweaks that might be worthwhile:

1) `s/include/includes/`
2) add "the" before `Access Log Extensions`

**Comment by i110** [comment]
@tmaesaka thank you for the comment, I addresses the issues

**Comment by kazuho** [comment]
@i110 @tmaesaka Thank you!!!

**Pull request merged**